package.json

```
@@ -3999,6 +3999,446 @@
+														"name": "Activate parent environment",
+														"icon": "root-folder-opened",
+														"type": "command",
+														"command": "language-julia.changeCurrentEnvironment"
+													},
+													{
+														"key": "m",
+														"name": "Choose module",
+														"icon": "symbol-variable",
+														"type": "command",
+														"command": "language-julia.chooseModule"
+													},
+													{
+														"key": "p",
+														"name": "Open package directory",
+														"icon": "new-folder",
+														"type": "command",
+														"command": "language-julia.openPackageDirectory"
+													},
+													{
+														"key": "t",
+														"name": "Tag new package version (experimental)",
+														"icon": "tag",
+														"type": "command",
+														"command": "language-julia.tagNewPackageVersion"
+													},
+												]
+											},
+											{
+												"key": "=",
+												"name": "+Format",
+												"icon": "list-flat",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "=",
+														"name": "Format region or buffer",
+														"icon": "list-flat",
+														"type": "command",
+														"command": "editor.action.format"
+													},
+													{
+														"key": "b",
+														"name": "Format buffer",
+														"icon": "file",
+														"type": "command",
+														"command": "editor.action.formatDocument"
+													},
+													{
+														"key": "c",
+														"name": "Format changes",
+														"icon": "diff",
+														"type": "command",
+														"command": "editor.action.formatChanges"
+													},
+													{
+														"key": "s",
+														"name": "Format selection",
+														"icon": "selection",
+														"type": "command",
+														"command": "editor.action.formatSelection"
+													},
+													{
+														"key": "B",
+														"name": "+Format buffer with formatter",
+														"icon": "file",
+														"type": "command",
+														"command": "editor.action.formatDocument.multiple"
+													},
+													{
+														"key": "S",
+														"name": "+Format selection with formatter",
+														"icon": "selection",
+														"type": "command",
+														"command": "editor.action.formatSelection.multiple"
+													}
+												]
+											},
+											{
+												"key": "b",
+												"name": "+Backend",
+												"icon": "circuit-board",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "i",
+														"icon": "output",
+														"type": "command",
+														"command": "language-julia.refreshLanguageServer",
+														"name": "Julia: Re-Index Language Server Cache",
```

```suggestion
														"name": "Julia: Re-Index Language Server Cache",
														"icon": "output",
														"type": "command",
														"command": "language-julia.refreshLanguageServer",
```
Nit: put the "name" under "key" to be consistent with the format of other bindings